.gitignore

```diff
@@ -2,3 +2,4 @@ vendor
 composer.phar
 composer.lock
 phpunit.xml
+/bin
```

Comment:
End of Line, plz, or the panda gets angry.

![](http://www.reactiongifs.com/wp-content/uploads/2012/10/f-this-panda.gif)